### DISCUSSION TITLE
Webrtc API endpoint, local connection

### DISCUSSION THREAD

**Discussion by gtxaspec** [mrlt8/docker-wyze-bridge#923] [General]
On the api's :5000/webrtc/cam endpoint, currently it uses kvs to fetch the stream.  It's there a way to establish a local connection directly to the camera using webrtc?

**Comment by mrlt8** [comment]
Not yet, but hopefully soon. It's still a work in progress.

**Comment by gtxaspec** [comment]
Alexxit from go2rtc has added support for docker-wyze-bridge's webrtc signaling endpoints, and my preliminary testing shows that,  when using both go2rtc +docker wyze bridge, the connections are indeed p2p, not using relay servers.

Check it out If you have some time. 
Just fyi!!

**Reply by baudneo** [reply]
Was just looking for info on how to have a local only webRTC stream. I am using wz-mini-hacks as the FW on my wyze cams and this bridge software works, just need to test self-hosted mode.

My goal is to have 2 way audio similarly to frigate card but in a generic camera card for HASS/ZoneMinder.

Thanks for your work @gtxaspec and @mrlt8, I really appreciate it!

**Comment by plat2on1** [comment]
> Alexxit from go2rtc has added support for docker-wyze-bridge's webrtc signaling endpoints, and my preliminary testing shows that, when using both go2rtc +docker wyze bridge, the connections are indeed p2p, not using relay servers.
> 
> Check it out I'd you have some time. Just fyi!!

Does this work with V3 cameras? The WebRTC link is always null for my cameras.
